### PR TITLE
COCICCS-6228

### PR DESCRIPTION
**Pull request by nithyal1**

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/BuildDetails.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// =========================================================================
+// Copyright 2020 T-Mobile, US
```
Comment (sarathmohan4) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/BuildDetails.java:2]: year

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/OIDCAuthService.java [hunk 2]
```diff
@@ -450,4 +458,39 @@
         }
         return ResponseEntity.status(HttpStatus.OK).body(groups);
     }
+
```
Comment (sarathmohan4) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/OIDCAuthService.java:461]: Can we move this to a more relavent/ new servcice ?

**Review by sarathmohan4 — CHANGES_REQUESTED** [review]

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/BuildDetails.java (hunk 1 above)
Comment (nithyal1) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/model/BuildDetails.java]: updated

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/OIDCAuthService.java (hunk 2 above)
Comment (nithyal1) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/OIDCAuthService.java]: moved to new service

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/OIDCAuthService.java [hunk 3]
```diff
@@ -450,4 +450,4 @@
         }
         return ResponseEntity.status(HttpStatus.OK).body(groups);
     }
-}
+}
```
Comment (sarathmohan4) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/OIDCAuthService.java:453]: can we undo this?

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/v2/controller/OIDCAuthController.java [hunk 4]
```diff
@@ -281,4 +281,4 @@
 		return oidcAuthService.searchGroupEmailInAzureAD(email);
 
 	}
-}
+}
```
Comment (sarathmohan4) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/v2/controller/OIDCAuthController.java:284]: can we undo this

**Review by sarathmohan4 — DISMISSED** [review]

**Review by alexrajan3 — APPROVED** [review]

**Pull request merged**